content/chainguard/libraries/faq.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Chainguard Libraries FAQ"
+linktitle: "FAQ"
+type: "article"
+description: "Frequently Asked Questions and answers for Chainguard Libraries users"
+draft: false
+menu:
+  docs:
+    parent: "libraries"
+weight: 003
+toc: true
+---
+
+
+## Frequently Asked Questions
+
+### Does Chainguard Libraries for Java include CVE remediation fixes?
+
+Short answer: 
+
+No. Libraries are built from source code in the secured and hardened Chainguard
+infrastructure. This eliminates any build and distribution stage
+vulnerabilities.
+
+More details:
+
+Chainguard can not patch Java libraries and create binaries with the same
+identifier because the complete behavior and API surface of every library
+affects the use. That use however is part of the application development of each
+customer. It varies widely and any change potentially creates incompatibilities,
+different behavior or even new security issues. 
+
+Chainguard collaborates with many upstream projects and can collaborate with
+customers to increase and accelerate the creation and adoption of fixes and the
+work towards new releases.
+
+Importantly [over 95% of all known vulnerable components have a fixed version
+available](https://www.sonatype.com/blog/are-unnecessary-vulnerabilities-polluting-your-software-supply-chain)
+and by adopting those newer versions in your application you can remediate most
+CVE. Chainguard Libraries for Java includes those newest versions and adds the
+build and distribution channel security.
```

Comment:
Are there any other FAQs?

Reply:
Not yet .. but the two coming ones I have so far are about the name Chibbies (to be determined if that goes public here or anywhere) and a longer one about software supply chain attacks including namespace, access, and typosquatting stuff - all with links to more info .. but thats a big topic that I will write up more after Assemble